cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/analyzer/OptimizationOptions.java

```diff
@@ -73,6 +73,15 @@ public OptimizationOptions(Set<String> excludedTopics,
                              boolean isTriggeredByGoalViolation,
                              Set<Integer> requestedDestinationBrokerIds,
                              boolean onlyMoveImmigrantReplicas) {
+    if (excludedTopics == null) {
```

Comment:
Nit: in another project of mine, I have the following util method since doing the null check is quite common.

```
  public static <T> T validateNotNull(T obj, String objName) {
    Objects.requireNonNull(obj, objName + " cannot be null.");
    return obj;
  }
```

Reply:
Sure, that sounds good. Created https://github.com/linkedin/cruise-control/issues/1387 to make this case globally rather than case by case.